cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -85,6 +85,7 @@ share
     epochNo             Word64 Maybe        sqltype=word31type
     slotNo              Word64 Maybe        sqltype=word63type
     epochSlotNo         Word64 Maybe        sqltype=word31type
+    epochHeight         Word64 Maybe        sqltype=word31type
```

Comment:
If we are to add this, we will need a documentation entry down below in this file.